pyproject.toml

```diff
@@ -11,7 +11,7 @@ include = [
 ]
 
```

Comment:
Also one of the changes here is causing this error when running pytest: (env) pmurdoch@PED-ML-133 amanuensis % pytest tests_endpoints/endpoints 
/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/_pytest/config/__init__.py:195: PluggyTeardownRaisedWarning: A plugin raised an exception during an old-style hookwrapper teardown.
Plugin: helpconfig, Hook: pytest_cmdline_parse
AssertionError: 
For more information see https://pluggy.readthedocs.io/en/stable/api_reference.html#pluggy.PluggyTeardownRaisedWarning
  return pluginmanager.hook.pytest_cmdline_parse(
Traceback (most recent call last):
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/bin/pytest", line 8, in <module>
    sys.exit(main())
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/_pytest/config/__init__.py", line 58, in main
    config = _prepareconfig(args, plugins)
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/_pytest/config/__init__.py", line 195, in _prepareconfig
    return pluginmanager.hook.pytest_cmdline_parse(
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/pluggy/_hooks.py", line 513, in __call__
    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/pluggy/_manager.py", line 120, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/pluggy/_callers.py", line 156, in _multicall
    teardown[0].send(outcome)
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/_pytest/helpconfig.py", line 93, in pytest_cmdline_parse
    config = outcome.get_result()
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/pluggy/_result.py", line 100, in get_result
    raise exc.with_traceback(exc.__traceback__)
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/pluggy/_callers.py", line 103, in _multicall
    res = hook_impl.function(*args)
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/_pytest/config/__init__.py", line 675, in pytest_cmdline_parse
    self.parse(args)
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/_pytest/config/__init__.py", line 845, in parse
    self._preparse(args, addopts=addopts)
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/_pytest/config/__init__.py", line 799, in _preparse
    self.pluginmanager.load_setuptools_entrypoints("pytest11")
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/pluggy/_manager.py", line 422, in load_setuptools_entrypoints
    self.register(plugin, name=ep.name)
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/_pytest/config/__init__.py", line 306, in register
    ret = super(PytestPluginManager, self).register(plugin, name)
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/pluggy/_manager.py", line 169, in register
    hook._maybe_apply_history(hookimpl)
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/pluggy/_hooks.py", line 581, in _maybe_apply_history
    res = self._hookexec(self.name, [method], kwargs, False)
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/pluggy/_manager.py", line 120, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/pluggy/_callers.py", line 139, in _multicall
    raise exception.with_traceback(exception.__traceback__)
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/pluggy/_callers.py", line 103, in _multicall
    res = hook_impl.function(*args)
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/setuptools/_vendor/typeguard/_pytest_plugin.py", line 53, in pytest_addoption
    add_ini_option("string")
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/setuptools/_vendor/typeguard/_pytest_plugin.py", line 22, in add_ini_option
    parser.addini(
  File "/Users/pmurdoch/Documents/PCDC/amanuensis/env/lib/python3.9/site-packages/_pytest/config/argparsing.py", line 130, in addini
    assert type in (None, "pathlist", "args", "linelist", "bool")
AssertionError